### PR TITLE
Feat/learn clients

### PR DESCRIPTION
**Pull request by michaelsena**
This PR refactors @oed's recent "nodes" file from the build section into a few different files:
- learn/clients
- run/nodes
- tools/community-nodes

And it also refactors the build/installation page to be simplified and link out to these other documents.

### CODE REVIEW

File: docs/run/nodes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+### Using an external IPFS node
+
+It's also possible to run a separate service for IPFS using the `@ceramicnetwork/ipfs-daemon` npm package. 
+
+
+## **Network connectivity**
+
+### Expose ports
+
+If you want your node to be discoverable to the rest of the Ceramic Network, make sure the appropriate ports used by IPFS and the Ceramic HTTP API are publicly accessible. For example, if you are behind a router you may need to set up port forwarding.
```
Comment (oed) [inline docs/run/nodes.md:44]: Mention that ceramic uses port 7007 by default.

**Review by oed — APPROVED** [review]
Looks good, feel free to merge after you address my comments.

File: docs/build/installation.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 !!! warning ""
     
     **Project status: Clay testnet is now live.**</br>`Clay` is a decentralized public network ready for experimental application development and testing, but you still may encounter a few issues.  It is the last major milestone before `Fire` mainnet, which is under development and will launch in late Q1 2021. Documents published on Clay will *not* be portable to Fire. Please [reach out on Discord](https://chat.ceramic.network) or [create an issue on Github](https://github.com/ceramicnetwork/js-ceramic/issues) to report any issues. Read the full announcement [here](https://blog.ceramic.network/ceramic-network-clay-testnet/){:target="_blank"}.
 
 
 ## **Clients**
-Ceramic is available in a variety of clients suited for different use cases. For optimal performance, it is recommended that you use the HTTP Client when building an application.
+Ceramic is available in a variety of clients suited for different use cases. For optimal performance and data availability, it is recommended that you use the HTTP Client when building an application. For a full overview of the available Ceramic clients and their tradeoffs, see [Clients](../learn/clients.md).
```
[the remark this answers is not in the document]
Comment (michaelsena) [reply docs/build/installation.md:10]: I'm working on a `data availability` page as part of the learn section. This describes the persistence model of Ceramic. Once it's done today I was going to change this to a link to that page, which should provide all the additional detail needed. @oed

File: docs/build/installation.md [hunk 3]
```diff
@@ -65,13 +65,11 @@ Setup your client within your project.
...
-        - Run your own node `https://yourEndpoint.com`: Provides write and read access to the Clay testnet. Running your own node allows you to persist documents and have full control, however this is process is not yet well documented. If you choose to run your own node, be sure to add your node to the [`peerlist`](https://github.com/ceramicnetwork/peerlist/blob/main/testnet-clay.json) by submitting a pull request. This allows other nodes to discover your node.
-        - LocalHost `https://localhost:7007`: Provides read access to the Clay testnet. Writes made to this local node will only be available to nodes in the [`peerlist`](https://github.com/ceramicnetwork/peerlist/blob/main/testnet-clay.json), but will not be available to other nodes on the network. Users need to first have a Ceramic daemon running locally using the CLI.
+    When using the HTTP API, you need to connect to a remote Ceramic node by passing its URL. For this, you have a few options:
+    - [Free community nodes](../tools/hosted-nodes/community-nodes.md)
+    - [Third-party hosting services](../tools/hosted-nodes/node-providers.md)
+    - [Run your own node](../run/nodes.md)
```
[the remark this answers is not in the document]
Comment (michaelsena) [reply docs/build/installation.md]: fixed

**Pull request merged**